packages/url_launcher/url_launcher_windows/README.md

```diff
@@ -0,0 +1,31 @@
+# url_launcher_windows
+
+The Windows implementation of [`url_launcher`][1].
+
+## Backward compatible 1.0.0 version is coming
+The plugin has reached a stable API, we guarantee that version `1.0.0` will be backward compatible with `0.0.y+z`. If you use
+url_launcher_windows directly, rather than as an implementation detail
+of `url_launcher`, please use `url_launcher_windows: '>=0.0.y+x <2.0.0'`
```

Comment:
can I ask why we're recommending this instead of putting the right version of the plugin as a dependency of url_launcher?

Reply:
answering my own question, I assume we're doing this as a temporary message until Windows hits alpha.

Reply:
To endorse a federated plugin (i.e., make it a dependency of the main plugin), the plugin already has to be published. So our flow for a new federated implementation is:
1. Make a PR for the federated implementation (e.g., foo_windows).
2. Land foo_windows.
3. Publish foo_windows.
4. Make a PR (against foo) to endorse the new federated implementation.
5. Land foo.
6. Publish foo.

This README covers steps 3 through 5, where this plugin exists in the wild as a published plugin but isn't endorsed. In step 6, the foo_windows readme gets updated to say that it's included automatically by foo. (Then that gets published as a minor, README-only change to foo_windows).

It's cumbersome, but the alternative (putting everything in one PR) wouldn't pass CI because the build of the main plugin would fail due to the federated implementation not having been published.